docs/book/src/configuration/intro.md

```diff
@@ -0,0 +1,4 @@
+Rama's configuration system is still in the works but this section aims to be a living document of Rama's configuration system. Specifically, this section showcases all the configuration options from squid-config that we do and do not support and provide examples where possible. As a rule of thumb, Rama aims to support the latest squid-config version (at the time of writing that is v7).
```

Comment:
FYI squid-config is an inspiration, not something we want to be compatible with. It's a very common technology used by many, so clearly they did things right with that config. However it's a very complex config spec due to how much it supports and does. This is honestly not something I feel comfortable taking on in the project to be compatible, especially as the kind of project is not even that similar.

So for now we can look, besides how the config works and is defined, also what kind of rules do we want for rama. And initially I would already be happy if we can let the user define what ports to bind though, rules to control certain middleware (e.g. firewall, ratelimit, and other such rules).

But over time what we have and what they have will probably diverge more and more. And pretty certain that even at the start it might already look a bit different for the minimal stuff that is supported at that point.

Reply:
That all said. I do appreciate your work, initiative and input here. It makes me think that perhaps this issue is a bit premature. Or perhaps it is not.